docs/src/main/asciidoc/kafka.adoc

```diff
@@ -419,7 +419,7 @@ First, you need to include the `quarkus-jsonb` extension (if you already use the
 ----
 <dependency>
     <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-jsonb</artifactId>
+    <artifactId>quarkus-resteasy-jsonb</artifactId>
```

Comment:
You need to also change the extension name in the sentence above: "First, you need to include the `quarkus-jsonb` extension".

Reply:
Oh, thanks, I am on it

Reply:
this is my first contribution to a OS project ever :dancer:

Reply:
just pushed the change